examples/certs/intermediate/ca-int-cert.pem

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
there should be an easy way to regenerate these chain certs for when getting close to expiration dates